### ISSUE TITLE
Web-based tool for matching up strings in the Word docs with EDN keys

### ISSUE THREAD

**Issue by adamfeldman** [LuminareMed/quickscreen-translations#1]
Thank you Luminare team for all your efforts on https://CheckForCorona.com!!

tl;dr: I built a tool to match up already-translated strings with their English counterpart. 

How it works: a translator for a given language logs into this tool, picks an English string, and then searches the provided Word doc for that language to find the matching already-translated text. The text is copy-pasted into a form in this tool, and saved. 

The tool keeps track of the mapping between strings, locales, and tempura's EDN keys. The tool exports JSON which can be trivially converted back into EDN.

--

From the README: "Since these are medical questions, the integrity of the translation is important and we must use exactly the official translation from the documents (where applicable).". 

**Problem:** I found myself unable to interpret the docs containing translated strings. There don't appear to be any points of reference in the docs that would enable matching a given string with its resource ID in the translation dictionaries (the EDN files).

It's possible that a person who speaks one of the translation languages could match a given string with its English counterpart; the English strings are already matched with the resource IDs, which are necessary to complete the translation process. That person does not need to be the same person who made the translations, but it would be helpful if they were.

That matching could be done in something as simple as an Excel spreadsheet, but it's not the ideal UI for the task. I quickly built a custom UI and database using https://www.knack.com ("low-code" app builder).

**The tool is complete ("v0.1"), and also able to be rapidly modified to meet translators' needs.**

The frontend is currently protected by a login gate. Anyone can visit and sign-up, but is unable to do anything until I approve their account (this is quick to adjust, for example to have a single shared password protecting the whole application). https://coronahelp.knack.com/translations.

I've already loaded in the EN and ES strings from this repo (via helper script EDN --> CSV). I tested Arabic round-trip from web UI input  to export from Knack, to make sure non-ASCII text wasn’t mangled by encoding weirdness — AFAICT worked correctly, via Knack’s JSON export.

I still need to write some code to convert the JSON back to the EDN required by tempura, but that's trivial.

@oakmac would love to demo the app if this looks like it may help.

Alternatively, a language-speaker could possibly use a commercial product to do the matching, if this were implemented: https://docs.transifex.com/live/introduction.

Homepage:
<img width="1607" alt="Screen Shot 2020-03-26 at 9 33 58 PM" src="https://user-images.githubusercontent.com/133385/77715459-a82e8a80-6fa9-11ea-9e53-f1ff4a69a289.png">

Detail/edit page for a single resource:
<img width="1607" alt="Screen Shot 2020-03-26 at 9 38 44 PM" src="https://user-images.githubusercontent.com/133385/77715683-415da100-6faa-11ea-98ec-f20f423cbbe4.png">

**Comment by oakmac** [comment]
Very cool. I wonder if we could convince the translators who work for City of Houston / Harris County to use this directly? I don't think we can expect they will have git / GitHub knowledge, but they might be able to use this. Thoughts on that?

**Comment by adamfeldman** [comment]
That's exactly what this was built to enable! 🙂

**Comment by adamfeldman** [comment]
Imported data from (currently unmerged) PR #4 into the tool. Looks like it worked fine.

**Comment by adamfeldman** [comment]
Manually updated tool with changes from eab9b0da3667b15adc3a2a17a700dc3be1dc34e5
